spec/integration/workflows/steps.js

```diff
@@ -26,21 +26,22 @@ Given('Delete popup is open', () => {
 });
 
 Given('the Workflow dropdown is open', () => {
-  click("Selected Workflow Name");
+  click('Selected Workflow Name');
 });
 
-Given('{string} workflow is selected', (workflow) => {
-  click("Selected Workflow Name");
-  click(`${workflow} workflow`);
+Then('Workflow appeared with name {string}', (name) => {
+  $('.selected-workflow button.mak').contains(name);
 });
 
-Then('Workflow appeared with name {string}', (name) => {
-  cy.get('.selected-workflow button.mak').contains(name);
+Then('all the steps are loaded', () => {
+	$('Step Icons').each($el => {
+		cy.wrap($el).invoke('attr', 'src').should('not.contain', 'images/icon-default.svg');
```

Comment:
Some Steps don't have an icon, in which case they will - even in their loading states - have the fallback default Step icon for them.

Reply:
@erosdome but we can decide how to use it and when. If we use it to steps that we know should have icons it should be OK. or do you have any other idea how can we measure it?

Reply:
Understood, and yep I can't think of a better way to detect it currently.